src/pages/guides/functions/cognito-trigger-lambda-dynamodb/q/platform/[platform].mdx

```diff
@@ -31,17 +31,19 @@ You can skip this part, if you already have a GraphQL API with an User model.
 amplify add api
 
 ? Please select from one of the below mentioned services: GraphQL
+? Here is the GraphQL API that we will create. Select a setting to edit or continue: Name
 ? Provide API name: myapi
```

Comment:
nit: Can we also use the same name ( `contactapi`) as mentioned in the other flows?

Reply:
Updated.